src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -54,41 +56,41 @@ def test_warning(device_info):
     with nidmm.Session(device_info['name']) as session:
         session.configure_measurement_digits(nidmm.Function._2_WIRE_RESISTANCE, 1e6, 3.5)
         if not session.simulate:
-           try:
-               print(session.read(1000)) # Assume nothing is connected to device, overrange!
-               assert False
-           except nidmm.Warning as w:
-               print(w)
-               pass
+            try:
+                print(session.read(1000))  # Assume nothing is connected to device, overrange!
+                assert False
+            except nidmm.Warning as w:
+                print(w)
+                pass
         else:
-           pytest.skip("Simulated")
+            pytest.skip("Simulated")
 
 
-def test_ViBoolean_attribute(device_info):
+def test_ViBoolean_attribute(device_info):  # noqa: N802
```

Comment:
in our code-generated methods, these become vi_boolean
Might as well do the same and avoid # noqa